### ISSUE TITLE
feruz: Incremental development, week 1

### ISSUE THREAD

**Issue by Feruzteame** [HackYourFutureBelgium/class-9-10#177]
# Wednesday Check-In

__Incremental development, week 1__

## Progress

What have i learned 
- branches
- semantic-layout
- flex-box
- what is Agile?

completed so far this week?
- almost done

## Blocked

A little bit difficult to use command line, I feel comfort with git-kraken.

## Next Steps

I will  work on how can i write or plan good user story

## Tip(s) of the week

[advanced css](https://www.youtube.com/watch?v=FZrHoAUkHpE)
[flex box](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)

**Comment by snicoll** [comment]
@Feruzteame we'll review command line in class. Is there any particular reason you had to use it? Anything I can help you with?

**Comment by Feruzteame** [comment]
ja, i mean to use a git. I see the student use git in visual studio, i try but i can not it makes me confuse, i use only git-kraken  to be productive.
